### DISCUSSION TITLE
2023-12-03 题库无更新

### DISCUSSION THREAD

**Discussion by MlgmXyysd** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#1] [General]
新注册账户，2023-12-01 题库仍然适用
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/2957cf59-48fa-43c5-95dd-e98860611e00)

**Comment by klcb2010** [comment]
关闭这个讨论吧？标题引起误解
